ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main__capi-periodic-4.16.yaml

```diff
@@ -0,0 +1,255 @@
+  latest:
+    candidate:
+      product: ocp
+      stream: ci
+      version: "4.16"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: e2e-openstack-fips
+  minimum_interval: 56h
+  steps:
+    cluster_profile: openstack-vexxhost
+    env:
+      FEATURE_GATES: ClusterAPIInstall=true
+      FEATURE_SET: CustomNoUpgrade
+      FIPS_ENABLED: "true"
+      TEST_SUITE: experimental/reliability/minimal
+    pre:
+    - chain: ipi-openstack-pre
+    - ref: fips-check
+    workflow: openshift-e2e-openstack-ipi
+- as: e2e-openstack-proxy
+  minimum_interval: 144h
+  steps:
+    cluster_profile: openstack-vh-mecha-central
+    env:
+      FEATURE_GATES: ClusterAPIInstall=true
+      FEATURE_SET: CustomNoUpgrade
+    workflow: openshift-e2e-openstack-proxy
+- as: e2e-openstack-externallb
+  minimum_interval: 168h
+  steps:
+    cluster_profile: openstack-vh-mecha-central
+    env:
+      FEATURE_GATES: ClusterAPIInstall=true
+      FEATURE_SET: CustomNoUpgrade
+    workflow: openshift-e2e-openstack-external-lb
+- as: e2e-openstack-nfv-intel
+  minimum_interval: 144h
+  steps:
+    cluster_profile: openstack-nfv
+    env:
+      FEATURE_GATES: ClusterAPIInstall=true
+      FEATURE_SET: CustomNoUpgrade
+    workflow: openshift-e2e-openstack-nfv-cgroupsv1
+- as: e2e-openstack-nfv-mellanox
+  minimum_interval: 144h
+  steps:
+    cluster_profile: openstack-nfv
+    env:
+      ADDITIONAL_WORKERS_NETWORKS: mellanox-dpdk
+      FEATURE_GATES: ClusterAPIInstall=true
+      FEATURE_SET: CustomNoUpgrade
+      OPENSTACK_DPDK_NETWORK: mellanox-dpdk
+      OPENSTACK_SRIOV_NETWORK: mellanox-sriov
+    workflow: openshift-e2e-openstack-nfv-cgroupsv1
+- as: e2e-openstack-additional-ipv6-network
+  minimum_interval: 144h
+  steps:
+    cluster_profile: openstack-hwoffload
+    env:
+      FEATURE_GATES: ClusterAPIInstall=true
+      FEATURE_SET: CustomNoUpgrade
+    workflow: openshift-e2e-openstack-ipv6
+- as: e2e-openstack-csi-manila
+  minimum_interval: 144h
+  steps:
+    cluster_profile: openstack-vh-mecha-central
+    env:
+      FEATURE_GATES: ClusterAPIInstall=true
+      FEATURE_SET: CustomNoUpgrade
+    workflow: openshift-e2e-openstack-csi-manila
+- as: e2e-openstack-csi-cinder
+  minimum_interval: 144h
+  steps:
+    cluster_profile: openstack-vh-mecha-central
+    env:
+      FEATURE_GATES: ClusterAPIInstall=true
+      FEATURE_SET: CustomNoUpgrade
+      TEST_SKIPS: should concurrently access the volume and restored snapshot from
+        pods on the same node
+    workflow: openshift-e2e-openstack-csi-cinder
+- as: e2e-openstack-ovn-parallel
+  minimum_interval: 48h
+  steps:
+    cluster_profile: openstack-vexxhost
+    env:
+      FEATURE_GATES: ClusterAPIInstall=true
+      FEATURE_SET: CustomNoUpgrade
+      ROTATE_CLOUD_CREDENTIALS: "true"
+      TEST_SUITE: openshift/conformance/parallel
+    workflow: openshift-e2e-openstack-ipi
+- as: e2e-openstack-ovn-serial
```

Comment:
if we already have `e2e-openstack-ovn-parallel` do we really need the serial one?
Is there such a big difference in the test suite that is ran?
I'm just trying to reducing that amount of jobs that will run.

Reply:
The two suites run different tests. In particular: the "serial" suite runs tests that are susceptible to break each other, hence they're run without parallelisation. If I remember correctly, the rationale for having two suites (rather than running all serial-only, then all parallelisable tests) is that they're simply too many.

Coming back in scope of this particular PR: I am doublig the intervals of the Terraform-based jobs at the same time that I add the CAPI-based ones. The idea is that we should end up with the same exact number of jobs (except those that run very seldom, which I have not touched).